fbgemm_gpu/codegen/embedding_forward_quantized_split_template.cu

```diff
@@ -505,14 +509,13 @@ __global__ __launch_bounds__(kMaxThreads) void int_nbit_split_embedding_codegen_
             // FIXME: __any_sync with mask isn't supported by HIP yet.
             // See https://fburl.com/fvy7j0lq for the similar context.
             // assert false here with https://fburl.com/pfm7enw2
-            assert(false);
-            if (__any(found)) {
+            if (__any_sync(subwarp_mask, found)) {
 #else
             if (__any_sync(subwarp_mask, found)) {
 #endif
                 break;
 #ifdef __HIP_PLATFORM_HCC__
-            } else if (__any(empty)) {
+            } else if (__any_sync(subwarp_mask, empty)) {
```

Comment:
Now both HIP and CUDA versions look the same (`__any_sync(subwarp_mask, empty`).

Can we remove this `#ifdef`?

Reply:
It's removed.